doc/user/content/ops/diagnosing-using-sql.md

```diff
@@ -136,6 +136,11 @@ group by mdo.id, mdo.name
 order by sum(mas.records) desc;
 ```
 
```

Comment:
I feel like we should note that the sql queries above show all arrangements in Materialize, including system arrangements, whereas the memory usages visualization tool shows only user-created arrangements, and it shows them by dataflow. 

@frankmcsherry do you think it would be useful if we changed the sql queries above to ones that include the dataflow id so that it is easier to correlate the results of the queries to the dataflows they come from?

Reply:
I'm not sure tbh. The original intent of these queries was to be stepping stones towards being self-sufficient; simple enough that folks can see what is happening. Perhaps what we should add is something spanning all these queries which shows off how to correlate them back to that information?

Reply:
Ok I can add a section on "I found a problematic operator! How do I figure out which dataflow it came from?" to this doc.  

I still think that we need to note that the memory visualization only shows user-created operators by dataflow to prevent confusion in the event that the operator with the most records is a system one.

Reply:
That sounds like a great section! I'm not sure what to say about the memory viz thing; it sounds specific to it though, vs something to mention here (I have an alt memory viz PR, and it reports all the things, for example).

Reply:
+1 to everything above!  I added some text to point out the difference between sql queries + memory viz tool

Reply:
Ideally this wouldn't be the place to have reference documentation about the memory visualizer. Do we have somewhere else that we describe what it presents?

Reply:
it's primarily documented elsewhere (`docs/ops/monitoring/#memory-usage-visualization`), but i wanted to be sure to link the two, since there's no linkage between these 2 docs currently, so it feels a bit disjoint.

The way i envision the user workflow is:

1. user notices they're using a lot of memory, stumbles upon this page (diagnosing using sql) to begin troubleshooting 
2. if they're still stuck / want other resources, we point them to the memory viz tool from this doc